imagebuildah/build.go

```diff
@@ -1816,8 +1818,8 @@ func (b *Executor) deleteSuccessfulIntermediateCtrs() error {
 }
 
 func (s *StageExecutor) EnsureContainerPath(path string) error {
-	targetPath := filepath.Join(s.mountPoint, path)
-	_, err := os.Lstat(targetPath)
+	targetPath, err := securejoin.SecureJoin(s.mountPoint, path)
+	_, err = os.Lstat(targetPath)
```

Comment:
os.Stat should also work now as securejoin returns resolved path. iirc Lstat was put earlier as targetPath could be a symlink.

Reply:
Thanks!  Yeah, we probably being too cautious there with the Lstat, but I think I'll keep it as is just in case.